src/test/java/com/checkout/apm/oxxo/OxxoClientImplTest.java

```diff
@@ -37,9 +38,6 @@ class OxxoClientImplTest {
     @Mock
     private SdkAuthorization authorization;
 
-    @Mock
-    private EmptyResponse emptyResponse;
```

Comment:
We use this Class in both tests, is it really better to mock it in each test instead of mocking it here?